src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -38,6 +42,8 @@
 import static com.minecolonies.api.util.constant.ColonyConstants.*;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of "static" imports in this class from 5 to the maximum allowed 4. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3030)